lighting_reporting/static/src/scss/styles.scss

```diff
@@ -242,21 +218,6 @@ div.footer {
     float: left;
 }
 
-// family description
-.nl-family-description {
-    margin-bottom: 36px;
-
-    p {
-        font-family: "Raleway";
-        font-size: 10px; // 7pt
-        line-height: 16px; // 10pt
-        text-align: justify;
-        text-justify: inter-word;
-        border: 0;
-        padding: 0;
-    }
-}
-
```

Comment:
Why did you remove this?

Reply:
This class was only used by the lighting_reporting_seo module, and I have moved it